src/Button/README.md

```diff
@@ -7,78 +7,125 @@ categories:
+      <Button variant="inverse-tertiary" className="mb-2 mb-sm-0">Tertiary</Button>
+    </Stack>
+)}
 ```
 
 ## Utility Buttons
+
 ```jsx live
-<>
-  <div className="mb-2">
-    <Button variant="success" className="mb-2 mb-sm-0">Success</Button>{' '}
-    <Button variant="danger" className="mb-2 mb-sm-0">Danger</Button>{' '}
-    <Button variant="outline-success" className="mb-2 mb-sm-0">Success</Button>{' '}
-    <Button variant="outline-danger" className="mb-2 mb-sm-0">Danger</Button>{' '}
-  </div>
-  <div className="mb-2">
-    <Button variant="link" className="mb-2 mb-sm-0">Link</Button>{' '}
-    <Button variant="light" className="mb-2 mb-sm-0">Light</Button>{' '} 
-    <Button variant="dark" className="mb-2 mb-sm-0">Dark</Button>{' '}
-    <Button variant="outline-light" className="mb-2 mb-sm-0">Light</Button>{' '}
-    <Button variant="outline-dark" className="mb-2 mb-sm-0">Dark</Button>
-  </div>
-</>
+() => {
+  const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth });
+
+  return (
+    <>
+      <Stack
+        className="mb-2"
+        gap={2}
+        direction={ isExtraSmall ? "vertical" : "horizontal" }
+      >
+        <Button variant="success" className="mb-2 mb-sm-0">Success</Button>
+        <Button variant="danger" className="mb-2 mb-sm-0">Danger</Button>
+        <Button variant="outline-success" className="mb-2 mb-sm-0">Success</Button>
+        <Button variant="outline-danger" className="mb-2 mb-sm-0">Danger</Button>
+      </Stack>
+      <Stack
+        gap={2}
+        direction={ isExtraSmall ? "vertical" : "horizontal" }
+      >
+        <Button variant="link" className="mb-2 mb-sm-0">Link</Button>
+        <Button variant="light" className="mb-2 mb-sm-0">Light</Button>
+        <Button variant="dark" className="mb-2 mb-sm-0">Dark</Button>
+        <Button variant="outline-light" className="mb-2 mb-sm-0">Light</Button>
+        <Button variant="outline-dark" className="mb-2 mb-sm-0">Dark</Button>
+      </Stack>
+    </>
+)}
 ```
 
 ## Size
 
 ```jsx live
-<>
-  <div className="mb-2">
-    <Button variant="primary" size="lg" className="mb-2 mb-sm-0">
-      Large button
-    </Button>{' '}
-    <Button variant="outline-primary" size="lg" className="mb-2 mb-sm-0">
-      Large button
-    </Button>
-  </div>
-  <div className="mb-2">
-    <Button variant="primary" size="sm" className="mb-2 mb-sm-0">
-      Small button
-    </Button>{' '}
-    <Button variant="outline-primary" size="sm" className="mb-2 mb-sm-0">
-      Small button
-    </Button>
-  </div>
-  <div className="mb-2">
-    <Button variant="link" size="inline" className="mb-2 mb-sm-0">Inline button</Button>
-    <Button variant="link" size="inline" className="mb-2 mb-sm-0">Inline button</Button>
-  </div>
-</>
+() => {
+  const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth });
+
+  return (
+    <>
+      <div className="mb-2">
```

Comment:
I think you can now remove `div`'s here because you wrap buttons with `Stack` now and it has the same `className`